test/GRAPHFILES

```diff
@@ -94,6 +94,7 @@ users/npadmana/twopt/twopt-paircount.graph
 memleaksfull.graph
 studies/jacobi/jacobi.graph
 library/standard/List/perf/listBenchmark1.graph
+io/corrado/qio_buf_optimization_perf/large_binary_io.graph
```

Comment:
It looks like you're adding this to the Performance suite.  If I remember right, that's for the performance team to track how key aspects of the language are doing so they don't have to look over every single graph each week.  Have you talked to them about if it's okay to add this test there?  Otherwise, I'd put it in the "Misc" category or make a new suite for IO benchmarks (I'd say the Standard Libraries suite but that looks like it tracks package modules ...)

Reply:
Ok that makes sense, I'll it move to misc.